README.md

```diff
@@ -1,56 +1,23 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Greetings Microverse
 
 > Description the project.
 
 
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+-css and html
 
 
```

Comment:
## Your PR Title :taco:
![pr-title](https://user-images.githubusercontent.com/93039359/185084079-ec203a48-15e7-47bc-ad94-d1633e24fe19.png)


- [ ] Your PR has a title that is not so descriptive enough. Because **Hello micro** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Setup Hello Microverse Project** for your PR title. :+1: